.travis.yml

```diff
@@ -0,0 +1,56 @@
+---
+language: node_js
+node_js:
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
+  - "4"
+
+sudo: false
+dist: trusty
+
+addons:
+  chrome: stable
+  firefox: latest
+
+cache:
+  directories:
+    - $HOME/.cache # includes bower's cache
+    - $HOME/.npm
+
+env:
+  - SCENARIO_GROUP=default
+  - SCENARIO_GROUP=channels
+  - SCENARIO_GROUP=releases
+  - SCENARIO_GROUP=allowed-failures
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env: SCENARIO_GROUP=allowed-failures
+
+before_install:
+  - npm config set spin false
+  - npm install -g npm@4
+  - npm --version
+  - npm install -g bower
+
+install:
+  - npm install
+  - bower install
+
+script:
+  # Usually, it's ok to finish the test scenario without reverting
+  #  to the addon's original dependency state, skipping "cleanup".
+  - if [ "$SCENARIO_GROUP" = "default" ]; then
+      node_modules/.bin/ember try:each --config-path=config/ember-try.js ;
```

Comment:
Might be easier to modify the SCENARIO_GROUP lines above to be more like:

```yaml
env:
  - SCENARIO_GROUP_CONFIG_PATH=config/ember-try.js
  - SCENARIO_GROUP_CONFIG_PATH=config/ember-try/channels.js
  - SCENARIO_GROUP_CONFIG_PATH=config/ember-try/releases.js
  - SCENARIO_GROUP_CONFIG_PATH=config/ember-try/allowed-failures.js

# ...snip...
script:
  - node_modules/.bin/ember try:each --config-path=$SCENARIO_GROUP_CONFIG_PATH
```

Reply:
I'd rather move away from that pattern completely but wanted feature parity with the CircleCI config for now

Reply:
haha, I like the pattern 😝

Reply:
quite often its faster to run 3 different `ember try` scenarios in a single CI build compared to the time of creating one CI build per version (because a ton of time is wasted on setup, when the tests themselves are fast)

Reply:
true, though I'd rather use the `versionCompat` (or whatever it was called) option of `ember-try`